lib/features/app/app.dart

```diff
@@ -49,6 +49,10 @@ class _AppState extends State<App> {
           locale: _localizations.firstOrNull,
           localizationsDelegates: _localizationsDelegates,
           supportedLocales: _localizations,
+          // TODO(init): Uncomment it or delete if you don't need it.
+          // builder: (context, child) {
+          //   return DownByMaintenanceWidget(child: child!);
+          // },
```

Comment:
Can you check if you have the latest version of main branch here? There is the following builder in the main branch right now.
https://github.com/surfstudio/surf-flutter-app-template/blob/61946edd29013b40f801ad359cb28347507a08ac/lib/features/app/app.dart#L52